packages/ui/src/ui/containers/ACL/RequestPermissions/AclColumnGroupControl/AclColumnGroupControl.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import {AclColumnGroup} from '../../../../utils/acl/acl-types';
+import {Checkbox, Flex, Select, SelectOption} from '@gravity-ui/uikit';
+
+export type Props = {
+    value?: AclColumnGroup['id'];
+    onChange: (value?: Props['value']) => void;
+    columnGroups?: Array<AclColumnGroup>;
+    disabled?: boolean;
+};
+
+export function AclColumnGroupControl({value, onChange, columnGroups, disabled}: Props) {
+    const [checked, setChecked] = React.useState(Boolean(value));
+
+    const options = React.useMemo(() => {
+        return columnGroups
```

Comment:
`return columnGroups?.reduce<SelectOption<AclColumnGroup>[]>((acc, data) => {
    if (!data.removed) {
        acc.push({
            value: data.id,
            content: data.name,
            data,
        });
    }
    return acc;
}, []);`

I suggest changing it to reduce. this will reduce the number of runs